### PR TITLE
feat: Add load transaction syscall

### PR DESCRIPTION
**Pull request by xxuejie**

### CODE REVIEW

File: script/src/syscalls/mod.rs [hunk 1]
```diff
@@ -730,22 +732,29 @@ mod tests {
         machine.set_register(A2, 0); // offset
         machine.set_register(A7, LOAD_TX_HASH_SYSCALL_NUMBER); // syscall number
 
-        let hash = blake2b_256(&data);
+        let transaction_view = TransactionBuilder::default()
+            .output_data(data.pack())
```
Comment (jjyr) [inline script/src/syscalls/mod.rs]: This is weird, smile like a bug.

Comment (jjyr) [reply script/src/syscalls/mod.rs]: Oh, I see.

Comment (xxuejie) [reply script/src/syscalls/mod.rs]: It's in test, just a way to generate different transactions in proptest

**Pull request merged**